### ISSUE TITLE
Release for ubuntu 20.04 - Focal Fossa ?

### ISSUE THREAD

**Issue by Arpafaucon** [matterhorn-chat/matterhorn#656]
Hi developpers, and thanks for that marvellous CLI client !

I was wondering what is the current recommended way to get matterhorn running on `ubuntu 20.04`.

## Using the ubuntu 18 version

We basically get the same issues described in #240.

Launching the binary from the ubuntu 18 release yield the following error message 
```
❯ ./matterhorn 
./matterhorn: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory
```

That seem to be fixed by a `apt install libtinfo5`, but then you get another one:
```
❯ ./matterhorn              
./matterhorn: error while loading shared libraries: libffi.so.6: cannot open shared object file: No such file or directory
```

And `apt` does not seem to find `libffi6` in the standard repositories.

I guess that, even if I find a solution for `libffi`, I'll run into every version mismatch between Ubuntu 18 and 20, so that does not seem to be a wise solution.

## using the (unofficial, I believe) snap release

In that case, the binary runs fine, but I got blocked when trying to make the snapped matterhorn accept our internal certificate.

---
TL;DR:

You're doing an amazing job, I am aware you have a limited time as we all humans do. Could you tell me if there is a trick to get `matterhorn` running in ubuntu 20 ? If there aren't, is there an ETA for including ubuntu 20.04 as a release target ?

**Comment by jtdaugherty** [comment]
> And apt does not seem to find libffi6 in the standard repositories.

For what it's worth, the Ubuntu 20.04 package is `libffi7`, and that wouldn't work to load `libffi6`. (In general a good way to find out what provides something like this is to use `apt-file search`.)

**Comment by jtdaugherty** [comment]
I've uploaded a Focal build of the latest release:

https://github.com/matterhorn-chat/matterhorn/releases/tag/50200.10.1

And we'll include Focal builds in our releases going forward.
